src/modules/BootCommon.rb

```diff
@@ -493,7 +493,7 @@ def getBootloaders
       end
       ret = [getLoaderType(false)]
       ret << "grub2" # grub2 everywhere
```

Comment:
this is wrong if aarch support only efi, so it should be
```
ret << "grub2" unless Arch.aarch64
```
otherwise user can choose grub2 itself instead of efi variant in UI.